packages/angular/package.json

```diff
@@ -10,8 +10,8 @@
     "LICENSE"
   ],
   "peerDependencies": {
-    "@angular/core": "^19.0.0",
-    "@angular/platform-browser-dynamic": "^19.0.0",
+    "@angular/core": "^20.0.0",
```

Comment:
Yeah, let's do this for now. I am still believing that - given it comes with almost zero costs - supporting down 19 or 18 should be something for the backlog.